crates/tabby-common/src/path.rs

```diff
@@ -53,4 +54,70 @@ pub fn events_dir() -> PathBuf {
     tabby_root().join("events")
 }
 
+fn normalize_path_for_platform(
+    filepath: Option<String>,
+    is_windows: bool,
```

Comment:
i believe not matter where the server binary runs (linux / windows), we shall always normalize the path (since client side can be either windows or unix)